docs/challenges/language-support/cpp.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: "C++"
+description: "Learn how to solve CodeCrafters challenges in C++"
+---
+
+# Local development
+
+To run the code for your challenge locally, you'll need `cmake` & `make` installed.
+
+The script for your challenge (like `./your_sqlite3.sh`) will automatically compile your code using the rules defined in `CMakeLists.txt` before executing it. It'll
+look something like this:
+
+```sh
+#!/bin/sh
+
+set -e
+cmake $(dirname $0) > /dev/null
+make > /dev/null
+exec ./server "$@"
+```
+
+# File structure
+
+- The files for your solution are placed in the `src/` directory.
+- `src/Server.cpp` contains the `main` function, which is what the test runner executes.
```

Comment:
Let's call this `src/Main.cpp` (we'll then change all challenges to use this instead).

Saying `src/Server.cpp` doesn't make sense for a challenge like Grep, for example.